src/main/java/dev/meyi/bn/utilities/Utils.java

```diff
@@ -208,27 +221,29 @@ public static void drawCenteredString(String text, int x, int y, int color, floa
     Minecraft.getMinecraft().fontRendererObj.drawString(text, x, y, color);
     GL11.glScalef((float) Math.pow(moduleScale, -1), (float) Math.pow(moduleScale, -1), 1);
   }
-
-  public static void initializeConfigValues() {
-    if (BazaarNotifier.config.has("craftingListLength")) {
-
-      Configuration.craftingListLength = BazaarNotifier.config.getInt("craftingListLength");
-      Configuration.suggestionListLength = BazaarNotifier.config.getInt("suggestionListLength");
-      Configuration.craftingSortingOption = BazaarNotifier.config.getInt("craftingSortingOption");
-      Configuration.showInstantSellProfit = BazaarNotifier.config
-          .getBoolean("showInstantSellProfit");
-      Configuration.showSellOfferProfit = BazaarNotifier.config.getBoolean("showSellOfferProfit");
-      Configuration.showProfitPerMil = BazaarNotifier.config.getBoolean("showProfitPerMil");
-      Configuration.collectionCheckDisabled = BazaarNotifier.config
-          .getBoolean("collectionChecking");
-    } else {
-      Configuration.craftingListLength = Defaults.CRAFTING_LIST_LENGTH;
-      Configuration.suggestionListLength = Defaults.SUGGESTION_LIST_LENGTH;
-      Configuration.craftingSortingOption = Defaults.CRAFTING_SORTING_OPTION;
-      Configuration.showInstantSellProfit = Defaults.INSTANT_SELL_PROFIT;
-      Configuration.showSellOfferProfit = Defaults.SELL_OFFER_PROFIT;
-      Configuration.showProfitPerMil = Defaults.PROFIT_PER_MIL;
-      Configuration.collectionCheckDisabled = Defaults.COLLECTION_CHECKING;
+  public static void updateResources() throws IOException{
+    String result;
+    HttpGet request;
+    HttpResponse response;
+    HttpClient client = HttpClientBuilder.create().build();
+    request = new HttpGet(
+            "https://raw.githubusercontent.com/Detlev1/BazaarNotifier/master/src/main/resources/resources.json");
+    response = client.execute(request);
+    result = IOUtils.toString(new BufferedReader(new InputStreamReader(response.getEntity().getContent())));
+    BazaarNotifier.config.resources =  new JsonParser().parse(result).getAsJsonObject();
+    BazaarNotifier.bazaarConv = jsonToBimap(BazaarNotifier.config.resources.getAsJsonObject("bazaarConversions"));
+    BazaarNotifier.enchantCraftingList =  BazaarNotifier.config.resources.getAsJsonObject("enchantCraftingList");
+  }
+  public static BiMap<String, String> jsonToBimap(JsonObject jsonObject){
+    BiMap<String, String> b = HashBiMap.create();
+    Set<Map.Entry<String, JsonElement>> entries = jsonObject.entrySet();
+    for (Map.Entry<String, JsonElement> entry: entries) {
+      try {
+        b.forcePut(entry.getKey(), jsonObject.get(entry.getKey()).getAsString());
```

Comment:
I know this boils down to an issue with two bazaar items having the same name right now, but once that's changed, we need to change this to be back to `put` instead of `forcePut`